index.html

```diff
@@ -0,0 +1,187 @@
+            <path d="M335.96 630.22C335.96 630.22 334.302 630.643 333.873 631.692L341.939 637.049C341.939 637.049 342.562 635.121 342.818 634.639C343.099 634.143 343.575 633.234 343.575 633.234L341.316 629.899C341.341 629.884 338.632 628.778 335.96 630.22Z" fill="white" stroke="#36B37E" stroke-width="0.289082" stroke-miterlimit="10"/>
+            <path d="M363.387 597.129C363.387 597.129 366.954 601.979 368.168 607.954C369.304 613.507 368.442 614.038 368.442 614.038L371.757 607.985C371.757 607.985 372.574 605.41 370.545 602.243C368.516 599.075 363.387 597.129 363.387 597.129Z" fill="#36B37E"/>
+            <path d="M342.843 634.626C342.843 634.626 340.281 635.103 339.419 635.634C338.557 636.165 338.293 636.285 338.293 636.285L340.474 637.462L341.964 637.036L342.843 634.626Z" fill="white" stroke="#36B37E" stroke-width="0.289082" stroke-miterlimit="10"/>
+            <path d="M344.131 636.452L343.78 638.157L348.776 644.151C348.776 644.151 349.393 644.296 351.437 643.383L352.284 642.827L345.293 635.748C345.307 635.773 344.299 635.587 344.131 636.452Z" fill="white" stroke="#36B37E" stroke-width="0.289082" stroke-miterlimit="10"/>
+            <path d="M344.131 636.452C344.131 636.452 348.16 638.632 348.077 638.546C347.994 638.46 345.799 635.121 345.799 635.121C345.799 635.121 344.086 635.276 344.131 636.452Z" fill="white" stroke="#36B37E" stroke-width="0.289082" stroke-miterlimit="10"/>
+            </g>
+            <defs>
+            <clipPath id="clip0_48_1090">
+            <rect width="128.874" height="147.139" fill="white" transform="translate(340.936 238.255) rotate(-15.2643)"/>
+            </clipPath>
+            <clipPath id="clip1_48_1090">
+            <rect width="58.2472" height="66.5024" fill="white" transform="translate(50.1346 520.003) rotate(152.88)"/>
+            </clipPath>
+            <clipPath id="clip2_48_1090">
+            <rect width="22.4882" height="25.6754" fill="white" transform="translate(41.6981 134.557) rotate(82.5164)"/>
+            </clipPath>
+            <clipPath id="clip3_48_1090">
+            <rect width="93.8566" height="62.9589" fill="white" transform="translate(375.31 108.234) rotate(-145.241)"/>
+            </clipPath>
+            <clipPath id="clip4_48_1090">
+            <rect width="20.4102" height="16.2672" fill="white" transform="translate(151.014 475.325) rotate(61.0741)"/>
+            </clipPath>
+            <clipPath id="clip5_48_1090">
+            <rect width="48.4212" height="38.5925" fill="white" transform="translate(322.564 615.689) rotate(-30)"/>
+            </clipPath>
+            </defs>
+            </svg>            
+    </section>
+    <section class="landing-page">
+        <div class="hi">
+            <h1>Hello there, I'm Joaquin</h1>
+            <h1 class="enphasis">I'm a software developer</h1>
```

Comment:
You implemented the `h1` tag for the main title. That's nice 😃. But it would be better to use `h2` or `h3` tag for the subtitle `I'm a software developer` to improve our structure and semantics